tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -14,6 +14,29 @@
     $(cat {{ edpm_privatekey_path }} | base64 | sed 's/^/        /')
     EOF
 
+- name: stop libvirt services
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+
+    ServicesToStop=("tripleo_nova_libvirt.target"
+                    "tripleo_nova_virtlogd_wrapper.service"
+                    "tripleo_nova_virtnodedevd.service"
+                    "tripleo_nova_virtproxyd.service"
+                    "tripleo_nova_virtqemud.service"
+                    "tripleo_nova_virtsecretd.service"
+                    "tripleo_nova_virtstoraged.service")
```

Comment:
it might make sense to also stop the nova_comptue agent

there should be a tripleo_* version of that